pkg/protos/scalapb/scalapb.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.33.0
-// 	protoc        v5.26.0
+// 	protoc        v3.21.12
```

Comment:
This is weird, did you use the dev container to run `mage generate`?
I would not have expected this to change.

Reply:
Yes, as far as I recall! 🤔